configFunctions.f90

```diff
@@ -91,21 +101,24 @@ subroutine findReactionsWithProductOrReactant( r, chs, arrayLen )
     return
   end subroutine findReactionsWithProductOrReactant
 
+  ! ----------------------------------------------------------------- !
```

Comment:
I use emacs, which has a fortran mode. I think emacs sets default line width to 72 for historical reasons. Personally I like a visual aid to identify the functions and subroutines. But all these things are a bit personal preferences and i am happy to change them to be more standard!